### ISSUE TITLE
Rename XACT P1/P2/D1/D2(TD) Parameters in Device Config to more logical names, add Help button and real-time Center point adjustment

### ISSUE THREAD

**Issue by frskyrc-ben** [FrSkyRC/ETHOS-Feedback-Community#4865]
Currently in Device Config - Servos - XACT, the parameters for configuration are pretty vague as they are named "P1", "P2", "D1" and "TD"

![image](https://github.com/user-attachments/assets/24a12809-558f-4d76-87b0-7253da2e4147)

A) Can we get these parameters renamed to more logical names :

![image](https://github.com/user-attachments/assets/0a75ab51-b139-430d-861a-335ff911ced3)

B) It would also be very useful to add a button for "Help" to each parameter with additional programming information and directions for the parameter adjustments.

For example :
What does changing the Locking Arm Strength (P1) do versus Locking Arm Strength (P2) ? Under what condition should the locking arm strength be adjusted ? I noticed when setting the Locking Arm Strength (P1) very high, the servo start to jitter, so then I backed this down to the point it does not jitter. Is this the correct method for this setting ? It should be set as high as possible without causing adverse jittery or chatter for the servo ?

What does changing the Friction, Filter or Pot Gap do and under what conditions/symptoms should these be adjusted ?

Many users have reported confusion about these values and asked for better details and documentation for tuning the servos and how to set these adjustments accordingly.

C) Lastly, when adjusting the servo "Center" value, is it possible to make the adjustment "Live" or real-time? Currently, you can only see the physical center position adjustment after setting the value and pressing "RTN" or clicking away from the field.

**Comment by Wolepo** [comment]
Yes, a clear explanation is necessary. Also, these sensors cannot be found in the i18n.

**Comment by EngelMT** [comment]
I agree, if we offer the customers the possibility to make settings off the servos it is important for the customers to understand what they do.

**Comment by pstasek** [comment]
The only description of the parameters I found is available here: https://www.airbossrc.com/post/servo-settings-for-bls5400-series-xact-servos 

Indeed, this should at least be part of the official documentation. It would be helpful to have the description available within Ethos.

**Comment by frskyrc-ben** [comment]
keep open

**Comment by dodds-steven** [comment]
Yes Please

**Comment by mawzthefinn** [comment]
Latest XAct Script now has this change.

**Comment by bsongis-frsky** [comment]
I am not sure that our translations match between the Lua script and Ethos 26.1.0
This needs to be checked before RC4

**Comment by spoke2570** [comment]
<img width="485" height="519" alt="Image" src="https://github.com/user-attachments/assets/22f7b7c5-7850-4341-b59c-16c8bc85bdb1" />

This is RC7. Def improved from where this all started with searching for obscure threads in RC forums for help, and ultimately using trial and error.

P1 and P2: These need  better definitions as of now they are same.

Friction: I assume to be duration to next stable pot position? (Damper?)

Filter: This seems to remove input jitter? is this not already handled in the hardware pane (analog filter on/off) to remove unwanted stick jitter and otherwise by user developed curves to produce a center dead band?

Pot Gap: What even is this when the filter/analog filter/user curves remove weird pot inputs ? 

These beat what was there before and I understand not letting the full diff EQ statements and solutions to the public but there is still little that users can glean from this.

**Comment by spoke2570** [comment]
Lots of error by the way. Crazy self-exciting control response from stock servos is expected.

**Comment by Nicholas-Luoyi** [comment]
<img width="800" height="480" alt="Image" src="https://github.com/user-attachments/assets/2ffd202f-e797-40e8-9e6b-8e7217aa985a" />

Here is another version. Do you have any ideas?

**Comment by bsongis** [comment]
Right, the feedback about RC7 is excellent, I am about to release 26.1.0
This is the last issue (only labels) which I would like is sorted ...

**Comment by Wolepo** [comment]
Excellent. These terms are meaningful

**Comment by spoke2570** [comment]
I like those terms as well. We did go from 6 values to 4. I like simpler but are we now missing something important? Also what is the range of values available for each line?

**Comment by Nicholas-Luoyi** [comment]
**Holding strength (Range: 0.4 ~ 1.5): Value of P1 divided by P2**
For adjusting the jitter caused by mismatched stength performance.

**Operation smoothing: Value of D1**
For adjusting the smoothness of the servo throughout its operation.

**Deadband: Value of Pot gap**
Compensates for jitter taht cannot be resolved within the strength setting range.

D2 will be ignored

**Comment by mawzthefinn** [comment]
Pot Gap should be renamed Deadband in the English UI, that's the accepted term for the setting.

**Comment by spoke2570** [comment]
I like these. I suppose getting that bit of description and an available value range for each into Lothar's manual will be important.

**Comment by bsongis-frsky** [comment]
The XAct configuration has moved to a Lua script. It is available in FrSky suite. Closing the issue

**Comment by spoke2570** [comment]
Almost there with this Lua:

1) In online simulator the configure handler issues this error at initial startup: `Lua::load(config.lua): should return a table`
-Once cleared things operate normally switching between models etc.  Hit the restart button and the error reissues. (seems OK in the Suite sim - edit - and on the live radio)

2) This lua's launch spot in the sensor menu Is now labeled "XAct" and should be "Configure" as it was before. (more clear label)

3) This lua appears in Device config sub-menu "Sensors" and should have its own hardware category named "XAct" as before since the big part of it is arranging the physical operation of a device and not just a sensor. (similar to ESC not being merely a sensor and having its own category)

**Comment by Nicholas-Luoyi** [comment]
2) I'm not very clear with this point.

3) You are right

**Comment by spoke2570** [comment]
<img width="650" height="385" alt="Image" src="https://github.com/user-attachments/assets/f3d2a0af-a122-4d74-b609-ad77674743f2" />

Previously the label here was "Configure". The user already knows they have XAct servos using FBus, so label "XAct" does not describe the item's purpose.
